### ISSUE TITLE
Ubuntu 21.10 wget URL not found!

### ISSUE THREAD

**Issue by khteh** [dotnet/docs#26908]
```
$ wget https://packages.microsoft.com/config/ubuntu/21.10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
--2021-11-10 16:31:10--  https://packages.microsoft.com/config/ubuntu/21.10/packages-microsoft-prod.deb
Resolving packages.microsoft.com (packages.microsoft.com)... 40.65.182.21
Connecting to packages.microsoft.com (packages.microsoft.com)|40.65.182.21|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-10 16:31:10 ERROR 404: Not Found.
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 80707602-f283-e796-d3b8-3287fbc011dc
* Version Independent ID: 1d732b15-9e64-633a-000c-ad96d9d3818a
* Content: [Install .NET on Ubuntu - .NET](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu)
* Content Source: [docs/core/install/linux-ubuntu.md](https://github.com/dotnet/docs/blob/main/docs/core/install/linux-ubuntu.md)
* Product: **dotnet**
* Technology: **dotnet-install**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

**Comment by mohamed-azhar** [comment]
Tried a while ago and found out that its not even listed in [Microsoft Packages Directory](https://packages.microsoft.com/ubuntu/)

**Comment by adegeo** [comment]
Thank you for reporting this issue. The documentation is being updated. .NET packages haven't been released for Ubuntu 21.10 but are in progress. Using the 21.04 repository works just fine.

**Comment by adegeo** [comment]
Duplicate of #26879

**Comment by khteh** [comment]
> Thank you for reporting this issue. The documentation is being updated. .NET packages haven't been released for Ubuntu 21.10 but are in progress. Using the 21.04 repository works just fine.

Any ETA for 21.10?
